### PR TITLE
UICR-112: Move Course buttons into Actions dropdown

### PR DESCRIPTION
**Pull request by doytch**
Take a couple o' buttons and shove 'em in a dropdown. Two minutes, right? 😉 

The last two days resulted in the following changes in this PR:
- The tests were all failing from the get-go because the browser used by Cypress (Electron v80) was failing to render FilterAccordions properly when they get expanded. Temporarily switched to using `--browser chrome` but eventually was able to upgrade to Cypress `6.4.0` which contains Electron v88 which renders properly.
- Fixed the existing tests to expect the new UI.
- Kept running into an issue with Cypress (both v4 and v6) where `cy.task('coverageReport')` would error out. Digging around led me to realize that sometimes `.nyc_output/` can hang around which confuses the coverage reporting. Documented this error in the docs and added an npm script to `rm -rf` the dir because that's probably more discoverable for devs than a section in the docs.
- Updated to [the patched version of `yakbak-proxy`](https://github.com/folio-org/yakbak-proxy/pull/13) that won't error out when generating request hashes.
- Regenerated tapes.
- Oh yeah, took two buttons and shoved 'em in a dropdown.

### CODE REVIEW

File: cypress/integration/courses/02-courses.js [hunk 1]
```diff
@@ -126,15 +126,15 @@ describe('ui-courses: course creation, editing and deletion', () => {
     it('adds instructors', () => {
       // No plugin available in unit test, so we have to enter by hand
       cy.contains('0 instructors')
-      cy.contains('Edit instructor').should('not.exist')
+      cy.get('[icon=edit]').should('not.exist')
```
Comment (MikeTaylor) [inline cypress/integration/courses/02-courses.js:129]: Out of interest, why this change? It's defined that tests run under the `en` locale, isn't it?

File: cypress/integration/courses/02-courses.js [hunk 2]
```diff
@@ -231,6 +232,7 @@ describe('ui-courses: course creation, editing and deletion', () => {
 
     it('deletes the reserves', () => {
       // Check we can't delete a course with reserves
+      cy.contains('button', 'Actions').click()
```
Comment (MikeTaylor) [inline cypress/integration/courses/02-courses.js:235]: I love how simple this change to the tests is!

File: src/components/Course.js [hunk 3]
```diff
@@ -29,37 +30,32 @@ class Course extends React.Component {
     mutator: PropTypes.object.isRequired,
   };
 
-  renderLastMenu = () => {
-    return (
-      <PaneMenu>
```
Comment (MikeTaylor) [inline src/components/Course.js:34]: I guess `<PaneMenu>` was never really needed?

**Review by MikeTaylor — APPROVED** [review]
I hope it's OK that I didn't review the changes in the tapes :-)

On another occasion, it might be nice to do the tape regeneration as a separate PR, so it's separated from authored changes. But that shouldn't delay us here. Let's get this bad boy merged.

File: src/components/Course.js (hunk 3 above)
Comment (doytch) [reply src/components/Course.js:34]: Ehhh, I think it probably was tbh. The `lastMenu` and `actionsMenu` have slightly different APIs I believe. `actionsMenu` renders the dropdown button itself so it doesn't need a "container" like `PaneMenu`.

File: cypress/integration/courses/02-courses.js (hunk 1 above)
Comment (doytch) [reply cypress/integration/courses/02-courses.js:129]: Because I forgot to change the tests when I changed the `Edit instructor` buttons a while back to just be the edit `IconButton`s 🤦

**Comment by doytch** [comment]
> On another occasion, it might be nice to do the tape regeneration as a separate PR, so it's separated from authored changes. But that shouldn't delay us here. Let's get this bad boy merged.

Yeah I wondered about that. The only issue with that is that the PR's CI tests will fail without the new tapes. :\
